pkg/trace/config/store.go

```diff
@@ -0,0 +1,114 @@
+package config
+
+import (
+	"context"
+	"errors"
+	"sync"
+
+	"github.com/DataDog/datadog-agent/pkg/config/remote/service"
+	"github.com/DataDog/datadog-agent/pkg/proto/pbgo"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// Store stores config provided by agent-core and returns
+// latest configs to tracers.
+type Store struct {
```

Comment:
```suggestion
type Subscriber struct {
```
I think this is more like a subscriber isn't it? It subscribes to a remote store, and keeps the latest state.